pkgs/development/compilers/ocaml/camlboot.nix

```diff
@@ -0,0 +1,57 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, enableParallelBuilding ? true
+, guile
+}:
+
+stdenv.mkDerivation rec {
+  name = "camlboot";
+
+  src = fetchFromGitHub {
+    owner = "ekdohibs";
+    repo = "camlboot";
+    rev = "b103dc52411d94c8df69bc8b7915dd43aa40f8e8";
+    sha256 = "JFj9C1HqHpUgbq6cFvIfd6mOPy67s2C9ZPWMqsrB4d0=";
+    fetchSubmodules = true;
```

Comment:
Why not use our ocaml source?

Reply:
The submodule needs some light patching, and lags a bit behind upstream ocaml development. I believe it's currently based on ocaml 4.07.